### ISSUE TITLE
.h-full not working on safari?

### ISSUE THREAD

**Issue by bentzio** [tailwindcss/discuss#226]
Hey there! Loving Tailwind so far...  I'm playing around with a super simple layout for a back-end application and noticed that Safari seems to ignore the .h-full class in the content section. Here's the test code on pastebin (not using github yet): https://pastebin.com/nc08EWYY - look for div id="problem-here"....

All other browsers seem to honor it fine (Firefox, Chrome and Chrome-based browsers like Brave and Opera). Only Safari seems to ignore it outright.

Thanks!

**Comment by bentzio** [comment]
Sorry - ignore that pastebin url... that features the wrong copy/paste code.... this one is correct: https://pastebin.com/uQQNNczs

**Comment by adamwathan** [comment]
Yeah `h-full` is quirky with full screen stuff. Better to use `min-h-screen` and use flexbox to make the content area grow:

https://jsfiddle.net/o7nsvabt/
